.github/workflows/php.yml

```diff
@@ -0,0 +1,52 @@
+name: PHP Composer
+
+on:
+  push:
+    branches: [ 2.x ]
+  pull_request:
+    branches: [ 2.x ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+    
+    strategy: 
+      fail-fast: false
+      matrix: 
+        php-versions: ['7.1','7.2','7.3', '7.4','8.0']
+        phpunit-versions: ['latest']
```

Comment:
Unused, the PHPUnit version is defined in composer.json, if that does not support 7.1 - 8.0 then the version range should be extended in composer.json and Composer is going to pick the right version based on the current PHP version.